module_catalogue/deployments/helm_releases.tf

```diff
@@ -2,13 +2,15 @@ locals {
   nginx_template = templatefile("${path.module}/helm_values/nginx.yaml", {
   })
   prometheus_template = templatefile("${path.module}/helm_values/prometheus.yaml", {
-    "fqdn" = var.fqdn
+    "fqdn"        = var.fqdn
+    "environment" = var.environment
```

Comment:
Does the `environment` variable get used anywhere?

Reply:
Yes in the prometheus.yaml.

-> nginx.ingress.kubernetes.io/router.tls.domains.n.main: "prometheus-thanos-sidecar.${environment}.civo.io"